diagnostic.js

```diff
@@ -41,7 +57,10 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.update (
+  {House: "Arryn"},
+  {$unset: {motto : ""}}
+dh.houses.remove( {House: “Stark”} ))
```

Comment:
I assume you meant `db` here. But luckily for you, this line would've removed the `House: Stark` object entirely from the collection, and would have made your solution wrong!